checks/snmp/cambium.rb

```diff
@@ -0,0 +1,27 @@
+module Intrigue
+    module Ident
+    module SnmpCheck
+    class Cambium < Intrigue::Ident::SnmpCheck::Base
+    
+      def generate_checks 
+        [
+          {
+            type: "fingerprint",
+            category: "application",
+            tags: ["SNMPServer", "Networking"],
+            vendor: "Cambium",
+            product: "ePMP 3000 AP",
+            references: [],
+            version: nil,
```

Comment:
you can drop attributes without a value. Not a must, but can clean up the check a bit